database-management/src/main/java/gov/cms/ab2d/databasemanagement/DatabaseUtil.java

```diff
@@ -17,10 +16,12 @@
 import java.util.Properties;
 
 public class DatabaseUtil {
-
     private DatabaseUtil() {
     }
 
+    public static final String CREATE_LAMBDA_SCHEMA_STATEMENT = "CREATE SCHEMA if not exists lambda";
+    public static final String CREATE_PUBLIC_SCHEMA_STATEMENT = "CREATE SCHEMA if not exists public";
+
     public static Connection getConnection() throws SQLException {
```

Comment:
Can we make the SQL exception throw customized messages lets say if the property is not found or something like that ?

Reply:
done

Reply:
done